teams/teams-ps/teams/Remove-CsTeamsShiftsConnection.md

```diff
@@ -0,0 +1,70 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
+Module Name: MicrosoftTeams
+title: Remove-CsTeamsShiftsConnection
+author: sofiamaniou
+ms.author: somaniou
+manager: stepfitz
+online version: https://learn.microsoft.com/powershell/module/teams/remove-csteamsshiftsconnection
+schema: 2.0.0
+---
+
+# Remove-CsTeamsShiftsConnection
+
+## SYNOPSIS
+
+This cmdlet deletes a connection.
```

Comment:
I think the type of connection should be mentioned... Shifts?

Reply:
Done. Thanks!